kubernetes/gaffer/values.yaml

```diff
@@ -38,16 +38,49 @@ accumulo:
   config:
     files: {}
     accumuloSite:
-      instance.secret: DEFAULT
-      trace.user: root
-      trace.token.property.password: password
+      instance.secret: ""
+      # trace.user: tracer
+      # trace.token.property.password: password
       tserver.memory.maps.native.enabled: true
       tserver.memory.maps.max: 80M
       tserver.cache.data.size: 7M
       tserver.cache.index.size: 16M
       tserver.total.mutation.queue.max: 8M
       tserver.sort.buffer.size: 20M
       tserver.walog.max.size: 100M
+    userManagement:
+      rootPassword: ""
+      # If accumulo.config.userManagement.revoke.enabled is:
+      # true - Any users or permissions not specified below will be removed
+      # false - Only missing users and permissions will be added
+      revoke:
+        enabled: true
+      users:
+        tracer:
+          password: ""
+          permissions:
+            system:
+            - CREATE_TABLE
+            table:
+              trace:
+              - READ
+              - WRITE
+        gaffer:
+          password: ""
+          auths: []
+          permissions:
+            system:
+            - CREATE_TABLE
+            table:
+              simpleGraph:
```

Comment:
What does this mean if the user wants to use a different graph table name? Do they have to explicitly give the gaffer user permissions on their graph?

Reply:
Yep, see [here](https://github.com/gchq/gaffer-docker/blob/c35d4556d808bf9a64ad75070255cd900ff4b7db/kubernetes/gaffer-road-traffic/values.yaml#L364) how the Road Traffic Helm Chart specifies permissions for the roadTraffic table.

Note that the code added in kubernetes/gaffer/templates/store-properties.yaml will make sure that the user hasn't forgotten to grant basic READ and WRITE permissions to the table that will be used.